doc/integrator/ngeo.rst.tmpl

```diff
@@ -6,7 +6,10 @@ ngeo
 Organization
 ------------
 
-The main page where we can redefine the header is in the file:
+Simple and advanced modes
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The main page where we can redefine an interface is in the file:
```

Comment:
This confuses me a bit
We are talking about the main page of a (user) interface. So is this about how to change the standard behavior of a GMF user interface?

Reply:
```suggestion
The HTML and the JS of an interface are in the files:
```

Reply:
OK thanks